runtime/services/catalog/artifacts/artifacts_test.go

```diff
@@ -137,23 +137,23 @@ timeseries: time
 smallest_time_grain: day
 default_time_range: P1D
 dimensions:
-- label: Dim0_L
-  property: dim0
-  description: Dim0_D
-- label: Dim1_L
-  property: dim1
-  description: Dim1_D
+    - label: Dim0_L
+      property: dim0
+      description: Dim0_D
+    - label: Dim1_L
+      property: dim1
+      description: Dim1_D
```

Comment:
yaml.v3 doesn't support setting indentation for sequences
https://github.com/go-yaml/yaml/issues/661